benchmarks/benchmarks/analysis/rdf.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import division, absolute_import, print_function
+
+import MDAnalysis
+
+try:
+    from MDAnalysisTests.datafiles import PSF, DCD
```

Comment:
This is a triclinic system, so I think the uptick in time taken is the fix we did recently where we have to compare all possible images and choose the minimum, rather than being able to know which image to use.  I'm not sure using a triclinic cell is representative of most users? Maybe it is more common than I think though

Reply:
Many people use rhombic dodecahedron, which is triclinic.

Reply:
However, more importantly, the RDF of a protein-only system is a bit weird ;-). And the PSF/DCD tests do not have box information.

The TPR/XTC combo is protein in water for 10 frames. Selecting the water should give you something to chew on... and it is rhombic dodecahedron IIRC.